Yasgui/packages/yasgui/src/Tab.ts

```diff
@@ -500,11 +500,12 @@ export class Tab extends EventEmitter {
     response: any,
     duration: number,
     queryStartedTime: number,
-    hasMorePages?: boolean
+    hasMorePages?: boolean,
+    atLeastElements?: number
```

Comment:
What does this property `atLeastElements` mean?

Reply:
All query are executed with a limit equal to page size + one. If the count of results is equal to page size + 1, it means that there is at least one more page and to retrieve all the result, we need to run count query. At this moment we know we have one more result, this variable holds elements that we know for sure. Until count query don't return total elements (or it can fail) result message info will be something like "Showing results from 0 to 35  of of at least {{atLeastElements}}".

Reply:
OK then, this should be just `leastElements`, `possibleElementsCount`, etc.